### ISSUE TITLE
[ALM Accelerator - BUG]: Solution Checker Error in Pipeline

### ISSUE THREAD

**Issue by nasilvae** [microsoft/coe-starter-kit#5193]
### Describe the issue

Unable to deploy solution. When inspecting the deploy job it shows that the solution checker is failing. However, when we ran the solution checker in the solution via the Power Platform solutions portal it does not show any errors.

![image](https://user-images.githubusercontent.com/95443860/229865918-74f513f4-1bf9-421d-bc59-d3c1b0316f82.png)

### Expected Behavior

Should be able to deploy solutions since solution checker in power apps does not show errors. 

### What component are you experiencing the issue with?

ALM Accelerator Pipelines

### What solution version are you using?

March 2023

### Steps To Reproduce

Deploy Solution using Canvas App

**Comment by mikefactorial** [comment]
@nasilvae based on the error message can you verify if the App Registration has the PowerApps-Advisor (Analysis All) permissions mentioned here https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-almacceleratorpowerplatform-preview#create-an-app-registration-in-your-azure-ad-environment?

**Comment by nasilvae** [comment]
> @nasilvae based on the error message can you verify if the App Registration has the PowerApps-Advisor (Analysis All) permissions mentioned here https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-almacceleratorpowerplatform-preview#create-an-app-registration-in-your-azure-ad-environment?

Hi yes it has the permissions. 

![image](https://user-images.githubusercontent.com/95443860/229885286-c7160b6a-1b96-43aa-8030-f19fee20b447.png)

**Comment by mikefactorial** [comment]
@nasilvae can you verify if an App User has been created for the App Registration in the target environments? There's a gap in the setup docs using the Project Wizard that we are working on a fix for but possible that's the issue. https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-almacceleratorpowerplatform#create-an-app-user-in-your-dataverse-environments

**Comment by nasilvae** [comment]
> @nasilvae can you verify if an App User has been created for the App Registration in the target environments? There's a gap in the setup docs using the Project Wizard that we are working on a fix for but possible that's the issue. https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-almacceleratorpowerplatform#create-an-app-user-in-your-dataverse-environments

@mikefactorial

Yes, we have created an app user account and gave it sys admin privileges. This account was created on 3 environments that are being referenced in the profile.

**Comment by nasilvae** [comment]
I have also re-registered the app via PowerShell and still the same results. Only details i have about the error are the following:
![image](https://user-images.githubusercontent.com/95443860/229918196-655e6666-1f6c-4dc8-8077-1b9c512ffc7c.png)

**Comment by nasilvae** [comment]
> Hmm so this is runnning in GCC?

it is GCC. Good Point let me check and will report back.

**Comment by mikefactorial** [comment]
@nasilvae we have a few open issues related to GCC so just a heads up sounds like you and @dpbnk are both working through GCC related issues would be great for all of us to collaborate on where the gaps may exist. https://github.com/microsoft/coe-starter-kit/issues/5211

**Comment by mikefactorial** [comment]
@nasilvae closing this out as there doesn't seem to be anything further to do here.

**Comment by GU-VITA** [comment]
> @nasilvae we have a few open issues related to GCC so just a heads up sounds like you and @dpbnk are both working through GCC related issues would be great for all of us to collaborate on where the gaps may exist. #5211

**Comment by GU-VITA** [comment]
Here are my GCC incidents
https://github.com/microsoft/coe-starter-kit/issues/5369
https://github.com/microsoft/coe-starter-kit/issues/5370

**Comment by GU-VITA** [comment]
@nasilvae,  I have posted the same issue #5369.  Just an FYI, since I am getting the same error when using the Power Platform Checker task from the Power Platform Build Tools (2.0.19) DevOps extension.

I am starting to suspect a firewall issue between Azure DevOps (commercial in our case) and a GCC endpoint power-apis-usgov001-public.azure-apihub.us

The Power Platform pipelines will be available to us by July 1st 2023 and I am hoping to use this solution as a temporary step.

In the meantime, I added the DisableSolutionChecker to the associated variable group and set the value to true.
